client/components/game-forms/EventForm.js

```diff
@@ -1,38 +1,39 @@
-/*global M */
```

Comment:
Don't we still need the global M or do we not use this EventForm in our edit event page